modules/policy/engine/src/test/java/org/eclipse/sw360/antenna/policy/engine/TestUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.policy.engine;
+
+import java.util.Collection;
+import java.util.Iterator;
+
+public class TestUtils {
+    /**
+     * Helper for getting the element at index x in a collection
+     *
+     * @param data The collection from which the element should be retrieved
+     * @param index The index of the element to be retrieved
+     * @param <T> The data type used in the collection
+     * @return The element found
+     * @throws IndexOutOfBoundsException If the index is bigger than the collections size
+     */
+    public static <T> T getIndex(final Collection<T> data, final int index) {
```

Comment:
Would not be necessary, if `Collection` would be a `List` ;)

Reply:
That is right, but sometimes you need a set and the interface changes :-).

If you need it in productive code, it might make sense to use a list, but using a list, so that your tests get around this, I prefer to have the tests a bit more complicated